### PR TITLE
Fix heartbeat

### PR DESCRIPTION
**Pull request by masell**
Fixed/Added heartbeat from SessionManager control for RawWebsocket Transport.
I think this transport should behave the same way as "normal" transport. Especially for the cleanup part.

`The acquired session can not be expired` <-- This is thus removed.

Oh, and btw, with this PR it also removes the _tick call on send. Which means now things can actually timeout if it does not receive any data from client. (I think that before they could never time out due to this.)

### CODE REVIEW

**Comment by asvetlov** [comment]
Please resolve conflicts

**Comment by masell** [comment]
@asvetlov Rebased.

**Comment by masell** [comment]
@asvetlov Fixed conflicts again.

**Comment by masell** [comment]
@asvetlov 
Please merge

File: sockjs/session.py [hunk 1]
```diff
@@ -261,7 +255,7 @@ class SessionManager(dict):
     _hb_task = None  # gc task
 
     def __init__(self, name, app, handler, loop,
-                 heartbeat=25.0, timeout=timedelta(seconds=5), debug=False):
+                 heartbeat=25.0, timeout=timedelta(seconds=40), debug=False):
```
Comment (asvetlov) [inline sockjs/session.py]: Why default change?

**Comment by masteralves** [comment]
Can we get this one merged without the default value change? We have a sockjs server that is not closing connections that stop responding to heartbeats. This would fix it.

**Comment by asvetlov** [comment]
#214 merged
If you want to change default timeout -- please make a separate request

**Pull request closed without merging**